controller/api/util/api_utils.go

```diff
@@ -116,6 +117,14 @@ type TapRequestParams struct {
 	LabelSelector string
 }
 
+// GatewayRequestParams conttains parameters that are used to build a
+// GatewayRequest
+type GatewayRequestParams struct {
+	RemoteClusterName string
```

Comment:
is this unused?

Reply:
it is used

Reply:
it is used in `handleAPIGateways` handler.

Reply:
can you show me exactly where in the `handleAPIGateways` handler?  I see it being set on line 446 but as far as I can tell, the `requestParams` object is only ever passed to BuildGatewayRequest which does not use the RemoteClusterName field.

Reply:
fixed it